test/functional/testlib.bash

```diff
@@ -1409,7 +1420,15 @@ create_third_party_repo() { #swupd_function
 
 	debug_msg "Creating 3rd-party repo $repo_name..."
 	create_version -r "$env_name" "$version" 0 "$format" "$repo_name"
-	create_bundle -n os-core -v "$version" -f /usr/lib/os-release:"$OS_RELEASE",/usr/share/defaults/swupd/format:"$FORMAT" -u "$repo_name" "$env_name"
+
+	# we need to create os-core which should include the os-release and Swupd_Root.pem
+	debug_msg "Creating os-core with Swupd_Root.pem and os-release..."
+	hashed_name=$(sudo "$SWUPD" hashdump --quiet "$TEST_ROOT_DIR"/Swupd_Root.pem)
```

Comment:
Could we a openssl bash utility to use this? Technically we shouldn't be using a tool in a test that is trying to test itself.

Reply:
Yeah, this is known limitation we have in our test library. When I initially wrote the test library I asked about this and I was told to just use hashdump cause it was very hard to implement something to do the same thing. I agreed because the risk seems to be small and really the only tests that get somehow affected by this are those for the hashdump itself, it was definitely not worth the effort (I still think it is not).

Anyway the testilb already uses hashdump all over the place for generating hashes for files, so even if we decided to try an implement this ourselves (which I don't think we ever will) it would be part of a different PR since it is unrelated work.

Reply:
Sounds good. Ya looks like we are doing something adhoc there.